azurelinuxagent/ga/remoteaccess.py

```diff
@@ -153,8 +150,8 @@ def add_user(self, username, encrypted_password, account_expiration):
             raise RemoteAccessError("Error adding user {0}. {1}".format(username, ustr(e)))
         try:
             prv_key = os.path.join(conf.get_lib_dir(), TRANSPORT_PRIVATE_CERT)
-            pwd = self.cryptUtil.decrypt_secret(encrypted_password, prv_key)
-            self.os_util.chpasswd(username, pwd, conf.get_password_cryptid(), conf.get_password_crypt_salt_len())
+            pawd = self.cryptUtil.decrypt_secret(encrypted_password, prv_key)
```

Comment:
since we are fixing this, could we use 'password' or something more descriptive than 'pawd'?

Reply:
Memory and code scanners will look for "password".  Given it isn't even used elsewhere we could just fold it right into the chpasswd call instead, if this would prevent a merge.